docker-compose.yml

```diff
@@ -12,6 +12,8 @@ consul:
     - 8302
     - 8400
     - 8500
+    dns:
+       - 127.0.0.1
```

Comment:
I didn't end up using this, but leaving it in anyway so I can make it part of future demos of Consul

To try later: can I set this to `consul` on other containers and depend on the link to make it work?